cluster/ci/config/roles.yaml

```diff
@@ -699,3 +699,63 @@ items:
     kind: ClusterRole
     apiGroup: rbac.authorization.k8s.io
     name: system:image-puller
+
+- kind: Group
```

Comment:
All this should be an admin resource under your `services/` dir

Reply:
My bad.  I forgot to go back and move those into the service.  They have been moved now.